service/internal/releaseversion/release_versions_test.go

```diff
@@ -0,0 +1,102 @@
+package releaseversion
+
+import (
+	"context"
+	"strconv"
+	"testing"
+
+	"github.com/giantswarm/operatorkit/controller/context/cachekeycontext"
+
+	"github.com/giantswarm/cluster-operator/service/internal/unittest"
+)
+
+func Test_Release_Cache(t *testing.T) {
+	testCases := []struct {
+		name             string
+		ctx              context.Context
+		appName          string
+		expectCaching    bool
+		expectAppVersion string
+	}{
+		{
+			name:             "case 0",
+			ctx:              cachekeycontext.NewContext(context.Background(), "1"),
+			appName:          "cert-operator",
+			expectCaching:    true,
+			expectAppVersion: "1.2.1",
+		},
```

Comment:
The tests break now when I go to the unit tests package and change the versions. The tests would be more resilient and more informative when we would configure the initial release version of the component we test. Then you see input/output and can make more sense out of what is going on.